### ISSUE TITLE
Remmina anytime disconnect smartcard in RDP

### ISSUE THREAD

**Issue by SergeyDjam** [FreeRDP/Remmina#1279]
Client - ROSA Linux Fresh 8.1/9
Remmina version - Remmina - Version 1.2.0-rcgit-20 (git rcgit-20)
Desktop environment - KDE
Connecting to Windows 8 
Connecting via RDP (RDP/TLS)
RDP issues only, freerdp version (xfreerdp --version): FreeRDP version 2.0.0-dev (git n/a)

Anytime disconnected smartcard (eToken/ruToken).

We support Remmina 1.2+ versions only.
Please do not report bugs occurring on older versions!

Please give us some details about the systems you are using:
* Client (OS name and version): 
* Remmina version (remmina --version): 
* Desktop environment (GNOME, Unity, KDE, ..): 
* Connecting to (OS and version): 
* Connecting via (RDP, VNC, ...): 
* RDP issues only, freerdp version (xfreerdp --version):

**Comment by antenore** [comment]
Hi Sergey, can you test with xfreerdp and let me know if you have the same issue?

**Comment by useralexey** [comment]
HI , I'm joining in, there's this issue with Remmon this version.

    Client (OS name and version):Linux Mint 18.2 (ubuntu 16.04.1) 32 bit
    Remmina version (remmina --version):1.2.0-rcgit-24-0~remmina201710252018.r73bfd1e.ddd444db~ubuntu16.04.1
    Desktop environment ():Cinnamon
    Connecting to (OS and version): 2008 r2
    Connecting via (RDP ):RDP
    RDP issues only, freerdp version (xfreerdp --version):libfreerdp-* 2.0.0~git201711171423-0+remmina201711181817.rb5d2578.d0f49046~ubuntu16.04.1

But if you run xfreerdp .... / smartcard: everything is normally redirected.
Looking forward to a solution..

**Comment by ic3d** [comment]
Hi @SergeyDjam @useralexey we are reviewing old open issues. Did you still experience this one? Thanks!

**Comment by SergeyDjam** [comment]
Yes, the problem exists. Now Remmina 1.2.0-rcgit.28. The problem is shown less often, but hasn't disappeared.

**Comment by SergeyDjam** [comment]
FreeRDP 2.0.0-RC2 + Remmina 1.2.0-rcgit.29 fix this bug

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @antenore @ic3d @SergeyDjam @useralexey
